src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -122,11 +121,11 @@ public void createTimeline(final String eventTypeName, final String storageId, f
      */
     public List<Timeline> getActiveTimelinesOrdered(final String eventType)
             throws InternalNakadiException, NoSuchEventTypeException {
-        final List<Timeline> timelines = eventTypeCache.getTimelines(eventType);
+        final List<Timeline> timelines = eventTypeCache.getTimelinesOrdered(eventType);
         if (timelines.isEmpty()) {
             return Collections.singletonList(getFakeTimeline(eventTypeCache.getEventType(eventType)));
         } else {
-            return timelines.stream().sorted(Comparator.comparing(Timeline::getOrder)).collect(Collectors.toList());
+            return timelines.stream().filter(t -> t.getSwitchedAt() != null).collect(Collectors.toList());
```

Comment:
This is a very good answer why you need a test for this method.
Currently this method is not tested and anyone can change it in any way and nobody will notice.